src/Symfony/Contracts/HttpClient/Test/HttpClientTestCase.php

```diff
@@ -763,4 +763,62 @@ public function testGzipBroken()
         $this->expectException(TransportExceptionInterface::class);
         $response->getContent();
     }
+
+    public function testJsonRegexBuffer(): void
+    {
+        $client = $this->getHttpClient(__FUNCTION__);
+        $response = $client->request('GET', 'http://localhost:8057', ['buffer' => '/([a-z]+\/json)/']);
+
+        $firstContent = $response->getContent();
+        $secondContent = $response->getContent();
+
+        $this->assertSame($firstContent, $secondContent);
+    }
+
+    /**
+     * @expectedException \Symfony\Component\HttpClient\Exception\TransportException
+     */
+    public function testFalseBuffer(): void
```

Comment:
this is already tested in `testNonBufferedGetRequest`

Reply:
Removed the test